docs/docusaurus/docs/core/run_validations/_choose_result_format/_result_verbosity_reference_table.md

```diff
@@ -0,0 +1,17 @@
+The following table lists the fields that can be found in the `result` dictionary of a Validation Result and the Result Format verbosity levels that return that field.
+
+| Fields within `result`                |BOOLEAN_ONLY    |BASIC           |SUMMARY         |COMPLETE        |
+----------------------------------------|----------------|----------------|----------------|-----------------
+|    element_count                      |no              |yes             |yes             |yes             |
```

Comment:
would using ✅ vs ❌ or something visual like that be easier than "yes" / "no"? I feel like that's the more common way I see these tables.